src/WebJobs.Script/Diagnostics/HealthChecks/HealthCheckTags.cs

```diff
@@ -8,30 +8,36 @@ namespace Microsoft.Azure.WebJobs.Script.Diagnostics.HealthChecks
     /// </summary>
     internal static class HealthCheckTags
     {
-        private const string Prefix = "azure.functions.";
+        private const string FuncPrefix = "azure.functions.";
+        private const string WebJobsPrefix = "azure.web_jobs.";
```

Comment:
nit: I think I prefer webjobs as one, `azure.web_jobs.storage` feels off

```suggestion
        private const string WebJobsPrefix = "azure.webjobs.";
```

Reply:
I have the same underscore pattern for `script_host` and `web_host`. In this naming convention, different words are either namespace separated _or_ underscore separated. I don't want to violate that and have a single-word like `webjobs`.

Reply:
If we're already doing this with the other "hosts", then I would prefer the consistancy here as well

Reply:
Totally agree on keeping things consistent. That said, *WebJobs* feels more like a proper name to me, kind of like *Functions*, rather than something we'd split with an underscore.

Using `webjobs` without the underscore keeps that product identity and matches how it’s usually referenced across Azure.

That said, happy to follow the consistency rule if that’s the preferred convention.

Reply:
Because "WebJobs" is the brand term, I believe `azure.webjobs.` is the more correct approach here.

Reply:
Switched to `webjobs`.